lib/Dialect/SPIRV/Serialization/Deserializer.cpp

```diff
@@ -321,6 +324,9 @@ LogicalResult Deserializer::processDecoration(ArrayRef<uint32_t> words) {
         opBuilder.getIdentifier(attrName),
         opBuilder.getI32IntegerAttr(static_cast<int32_t>(words[2])));
     break;
+  case spirv::Decoration::ArrayStride:
+    typeDecorations[words[0]] = static_cast<uint64_t>(words[2]);
```

Comment:
Could you add a check to see that words.size() == 3 (like it is done for the other decoration types.

Reply:
Thanks!